templates/install_vars.j2

```diff
@@ -3,6 +3,7 @@ admin_password: 'password'
 pg_password: 'password'
 pg_database: 'awx'
 pg_username: 'awx'
+receptor_log_level: {{ receptor_log_level|default("debug") }}
```

Comment:
Is `debug` the default here?

Reply:
yes

Reply:
isnt the default "info" ?
```
[root@perfscale-automation-controller-control1-vsi ~]# cat /etc/receptor/receptor.conf | grep -i log
# Log Level
- log-level: info
```
Or did you mean we should just default to debug no mater what ?

Reply:
I do not know what is the factory-state default, but I just meant non-default value should come from config, not from config file, so if I do not care and not have it in the config, I get setup that is as much  "normal" as possible.

But still, this is a small thing :)

Reply:
Oh yeah, the "factory" default is `info`. I want the ability to SET it to debug. I could do
`receptor_log_level: {{ receptor_log_level|default("info") }}`